pkgs/applications/science/logic/abc/libabc.nix

```diff
@@ -30,6 +30,6 @@ stdenv.mkDerivation rec {
     homepage    = "https://people.eecs.berkeley.edu/~alanmi/abc/abc.htm";
     license     = stdenv.lib.licenses.mit;
     platforms   = stdenv.lib.platforms.unix;
-    maintainers = [ stdenv.lib.maintainers.thoughtpolice ];
+    maintainers = [ ];
```

Comment:
I'm not completely sure of the implications or criteria for package maintenance or the qualification process for maintainers, but I'm tentatively willing to under the assumption that it mostly involves keeping the package up to date and responding to nix-build-related issues for the package.